<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlGenerate</class>
 <widget class="QDialog" name="DlGenerate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>321</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 12pt &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabRandom">
    <attribute name="title">
     <string>随机生成</string>
    </attribute>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>进程数目</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="lineEditCount">
         <property name="styleSheet">
          <string notr="true">
font: 12pt &quot;微软雅黑&quot;;
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 2px 1px 2px; 
    min-width: 9em;  
comboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 10px;
 
    font: 12pt &quot;微软雅黑&quot;;
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; 
    border-top-right-radius: 3px; 
    border-bottom-right-radius: 3px;
}</string>
         </property>
         <property name="currentText">
          <string>10</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="maxCount">
          <number>100</number>
         </property>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tabManual">
    <attribute name="title">
     <string>手动生成</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">  font: 12pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>进程名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditName">
         <property name="styleSheet">
          <string notr="true">color: blue;
                        background-color: yellow;
                        selection-color: yellow;
                        selection-background-color: blue</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">  font: 12pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>到达时间</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditArrival">
         <property name="styleSheet">
          <string notr="true">color: blue;
                        background-color: yellow;
                        selection-color: yellow;
                        selection-background-color: blue</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">  font: 12pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>服务时间</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditServeTime">
         <property name="styleSheet">
          <string notr="true">color: blue;
                        background-color: yellow;
                        selection-color: yellow;
                        selection-background-color: blue</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">  font: 12pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>优先级</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditPriority">
         <property name="styleSheet">
          <string notr="true">color: blue;
                        background-color: yellow;
                        selection-color: yellow;
                        selection-background-color: blue</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>158</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QPushButton" name="pushButtonOK">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
